<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>BoxUi</class>
 <widget class="QDialog" name="BoxUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="2" >
    <widget class="LengthCombo" name="heightUnitsLC" />
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QComboBox" name="innerBoxCO" >
     <property name="toolTip" >
      <string>Inner box -- needed for fixed width &amp; line breaks</string>
     </property>
     <item>
      <property name="text" >
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Parbox</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Minipage</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QComboBox" name="typeCO" >
     <property name="toolTip" >
      <string>Supported box types</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="innerBoxLA" >
     <property name="toolTip" >
      <string>Inner box -- needed for fixed width &amp; line breaks</string>
     </property>
     <property name="text" >
      <string>Inner Bo&amp;x:</string>
     </property>
     <property name="buddy" >
      <cstring>innerBoxCO</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="typeLA" >
     <property name="toolTip" >
      <string>Supported box types</string>
     </property>
     <property name="text" >
      <string>&amp;Decoration:</string>
     </property>
     <property name="buddy" >
      <cstring>typeCO</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="heightED" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>1</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Height value</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="widthED" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>1</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Width value</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="heightLA" >
     <property name="toolTip" >
      <string>Height value</string>
     </property>
     <property name="text" >
      <string>&amp;Height:</string>
     </property>
     <property name="buddy" >
      <cstring>heightED</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="LengthCombo" name="widthUnitsLC" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="widthLA" >
     <property name="toolTip" >
      <string>Width value</string>
     </property>
     <property name="text" >
      <string>&amp;Width:</string>
     </property>
     <property name="buddy" >
      <cstring>widthED</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Alignment</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="2" >
       <widget class="QComboBox" name="halignCO" >
        <property name="toolTip" >
         <string>Horizontal alignment of the content inside the box</string>
        </property>
        <item>
         <property name="text" >
          <string>Left</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Center</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Right</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Stretch</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="halignLA" >
        <property name="toolTip" >
         <string>Horizontal alignment of the content inside the box</string>
        </property>
        <property name="text" >
         <string>Horizontal</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QComboBox" name="valignCO" >
        <property name="toolTip" >
         <string>Vertical alignment of the box (with regard to baseline)</string>
        </property>
        <item>
         <property name="text" >
          <string>Top</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Middle</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Bottom</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="valignLA" >
        <property name="toolTip" >
         <string>Vertical alignment of the box (with regard to baseline)</string>
        </property>
        <property name="text" >
         <string>&amp;Box:</string>
        </property>
        <property name="buddy" >
         <cstring>valignCO</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Co&amp;ntent:</string>
        </property>
        <property name="buddy" >
         <cstring>ialignCO</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="ialignCO" >
        <property name="toolTip" >
         <string>Vertical alignment of the content inside the box</string>
        </property>
        <item>
         <property name="text" >
          <string>Top</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Middle</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Bottom</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Stretch</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="ialignLA" >
        <property name="toolTip" >
         <string>Vertical alignment of the content inside the box</string>
        </property>
        <property name="text" >
         <string>Vertical</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="restorePB" >
       <property name="toolTip" >
        <string/>
       </property>
       <property name="text" >
        <string>&amp;Restore</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPB" >
       <property name="toolTip" >
        <string/>
       </property>
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyPB" >
       <property name="toolTip" >
        <string/>
       </property>
       <property name="text" >
        <string>&amp;Apply</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePB" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <includes>
  <include location="local" >qt_helpers.h</include>
 </includes>
 <customwidgets>
  <customwidget>
   <class>LengthCombo</class>
   <extends>QComboBox</extends>
   <header>LengthCombo.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>typeCO</tabstop>
  <tabstop>innerBoxCO</tabstop>
  <tabstop>widthED</tabstop>
  <tabstop>widthUnitsLC</tabstop>
  <tabstop>heightED</tabstop>
  <tabstop>heightUnitsLC</tabstop>
  <tabstop>ialignCO</tabstop>
  <tabstop>halignCO</tabstop>
  <tabstop>valignCO</tabstop>
  <tabstop>restorePB</tabstop>
  <tabstop>okPB</tabstop>
  <tabstop>closePB</tabstop>
  <tabstop>applyPB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
